### PR TITLE
Use new macros for BLT TPL exports

### PR DESCRIPTION
**Pull request by davidbeckingsale**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -106,7 +106,7 @@ RUN . /opt/spack/share/spack/setup-env.sh && spack load hip llvm-amdgpu && \
     cmake -DENABLE_WARNINGS_AS_ERRORS=Off -DCMAKE_CXX_COMPILER=amdclang++ -DUMPIRE_ENABLE_DEVELOPER_DEFAULTS=On -DENABLE_HIP=On .. && \
     make -j 16 VERBOSE=1
 
-FROM ghcr.io/rse-ops/hip-ubuntu-22.04:hip-4.3.1 AS hip.debug
+FROM ghcr.io/rse-ops/hip-ubuntu-20.04:hip-5.1.3 AS hip.debug
```
Comment (kab163) [inline Dockerfile:109]: The docker build seems to still be trying to find that path with hip-4.3.1

Comment (kab163) [reply Dockerfile:109]:
```
CMake Error in tests/integration/CMakeLists.txt:
#8 5.524   Imported target "blt_hip" includes non-existent path
#8 5.524 
#8 5.524     "/opt/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/hip-4.3.1-yx5zq4hxhqjl6zth4h7u46f2x4cl6ecp/../include"
#8 5.524 
#8 5.524   in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
#8 5.524 
#8 5.524   * The path was deleted, renamed, or moved to another location.
#8 5.524 
#8 5.524   * An install or uninstall procedure did not complete successfully.
#8 5.524 
#8 5.524   * The installation package was faulty and references files it does not
#8 5.524   provide.
```

Comment (adrienbernede) [reply Dockerfile:109]: @kab163 the change here affects the `hip.debug` docker target. However, Azure is set to run the `hip` target defined above which still uses hip 4.3.1.

Comment (adrienbernede) [reply Dockerfile:109]: It looks like an oversight on line 100.

Comment (adrienbernede) [reply Dockerfile:109]: However, whatever the version of hip, this path dose not look right:
```
/opt/spack/opt/spack/linux-ubuntu22.04-x86_64/gcc-11.2.0/hip-4.3.1-yx5zq4hxhqjl6zth4h7u46f2x4cl6ecp/../include
```
There is no include directory at the root of packages install directories.

**Comment by kab163** [comment]
If this is good to go, then I will merge. David is out this week, but he said last week that it was pretty much ready to go after it passed CI

**Comment by rhornung67** [comment]
Will this be the release candidate for v2024.02.0, effectively?

**Comment by davidbeckingsale** [comment]
Yes.

**Review by kab163 — APPROVED** [review]

**Pull request merged**